engine/access/rest/converter.go

```diff
@@ -0,0 +1,69 @@
+package rest
+
+import (
+	"fmt"
+
+	"github.com/onflow/flow-go/engine/access/rest/generated"
+	"github.com/onflow/flow-go/model/flow"
+)
+
+// Converter provides the conversion function to/from the Swagger object to Flow objects
+
+func toBlock(flowBlock *flow.Block) *generated.Block {
+	return &generated.Block{
+		Header:  toBlockHeader(flowBlock.Header),
+		Payload: toBlockPayload(flowBlock.Payload),
+	}
+}
+
+func toBlockHeader(flowHeader *flow.Header) *generated.BlockHeader {
+	return &generated.BlockHeader{
+		Id:                   flowHeader.ID().String(),
+		ParentId:             flowHeader.ParentID.String(),
+		Height:               fmt.Sprint(flowHeader.Height),
+		Timestamp:            flowHeader.Timestamp,
+		ParentVoterSignature: fmt.Sprint(flowHeader.ParentVoterSigData),
+	}
+}
+
+func toBlockPayload(flowPayload *flow.Payload) *generated.BlockPayload {
+	return &generated.BlockPayload{
+		CollectionGuarantees: collectionGuarantees(flowPayload.Guarantees),
+		BlockSeals:           blockSeals(flowPayload.Seals),
+	}
+}
+
+func collectionGuarantees(flowCollGuarantee []*flow.CollectionGuarantee) []generated.CollectionGuarantee {
+	collectionGuarantees := make([]generated.CollectionGuarantee, len(flowCollGuarantee))
+	for i, flowCollGuarantee := range flowCollGuarantee {
+		collectionGuarantees[i] = collectionGuarantee(flowCollGuarantee)
+	}
+	return collectionGuarantees
+}
+
+func collectionGuarantee(flowCollGuarantee *flow.CollectionGuarantee) generated.CollectionGuarantee {
+	signerIDs := make([]string, len(flowCollGuarantee.SignerIDs))
+	for i, signerID := range flowCollGuarantee.SignerIDs {
+		signerIDs[i] = signerID.String()
+	}
+	return generated.CollectionGuarantee{
+		CollectionId: flowCollGuarantee.CollectionID.String(),
+		SignerIds:    signerIDs,
+		Signature:    flowCollGuarantee.Signature.String(),
```

Comment:
This is incorrect, the golang implementation returns hex string: https://github.com/onflow/flow-go/blob/e533ad132d0ebe8af14315bd758f9061d2299fcc/crypto/sign.go#L130

But the API spec specifies that it should be base64: https://github.com/onflow/flow/blob/eeac4bb7b9a4a92fb6b9219bb1d5dba27b811e01/openapi/access.yaml#L803